### PR TITLE
MH-87: add collections page

### PR DESCRIPTION
**Pull request by nickghuman**
Created collections page

### CODE REVIEW

File: frontend/pages/collections/index.page.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            <CardContent>{collection.title}</CardContent>
+          </Card>
+        </a>
+      </Link>
+    </Grid>
+  ));
+  return (
+    <Layout>
+      <NextSeo
+        title={"Occurrences"}
```
Comment (iacobfred) [inline frontend/pages/collections/index.page.tsx:38]: @nickghuman, can you update this to `"Collections"`?

File: frontend/pages/collections/index.page.tsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        canonical={"/occurrences"}
+        description={
+          "Browse historical occurrences related to your topics or entities of interest."
```
Comment (iacobfred) [inline frontend/pages/collections/index.page.tsx:41]: How about `"Browse collections of historical occurrences, entities, sources, and more related to your topics of interest."` ?

File: frontend/pages/collections/[slug].page.tsx [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+import { GetStaticPaths, GetStaticProps } from "next";
+import { NextSeo } from "next-seo";
+import { FC } from "react";
+import { Container } from "react-bootstrap";
+interface CollectionProps {
+  collection: Collection;
+}
+
+/**
+ * A page that renders the HTML of a single entity.
```
Comment (iacobfred) [inline frontend/pages/collections/[slug].page.tsx:16]: --> 
```
* The collections page:
* http://www.modularhistory.com/collections
```

**Review by iacobfred — CHANGES_REQUESTED** [review]

**Review by iacobfred — APPROVED** [review]

**Review by mattfredericksen — APPROVED** [review]
LGM

**Pull request merged**